modulefiles/build.hera

```diff
@@ -7,20 +7,25 @@ module load cmake/3.16.1
 module load intel/18.0.5.274
 module load impi/2018.0.4
 
-module use -a /scratch1/NCEPDEV/stmp2/Rahul.Mahajan/test/UFS/modulefiles/nceplibs
+module use -a /scratch2/NCEPDEV/nwprod/NCEPLIBS/cmake/install/NCEPLIBS/modules
 
-module load w3nco
-module load w3emc
-module load nemsio
-module load bacio
-module load sp
-module load ip
-module load sfcio
-module load sigio
-module load gfsio
-module load nemsiogfs
-module load landsfcutil
-module load g2
+module load bacio/2.4.0
+module load crtm_dev/2.3.0
+module load g2/3.4.0
+module load g2tmpl/1.9.0
+module load ip/3.3.0
+module load nceppost/dceca26
```

Comment:
I think you can remove the load of crtm_dev and nceppost.  And what is g2tmpl?  Is that needed for g2/3.4.0?

Reply:
done.